src/DynamicData/List/ObservableListEx.cs

```diff
@@ -2120,6 +2120,19 @@ public static IObservable<IChangeSet<T>> Or<T>([NotNull] this ICollection<IObser
             return sources.Combine(CombineOperator.Or);
         }
 
+        /// <summary>
```

Comment:
These functions are needed, as the two functions above them collide now that ISourceList implements them both.  This function is a clarifying function that the compiler can use to remove the ambiguity.  It internally calls the IObservableList version instead of the ICollection version.